test/ImageTest.js

```diff
@@ -99,6 +99,14 @@ describe('Image', () => {
     tag.setProps({responsive: true});
     expect(tag.find('img').prop('data-src')).to.equal('http://res.cloudinary.com/demo/image/upload/sample');
   });
+  it('should update on prop change', () => {
+    let tag = mount(<Image publicId="sample" cloudName="demo"/>);
+    expect(tag.find('img').prop('src')).to.equal('http://res.cloudinary.com/demo/image/upload/sample');
```

Comment:
Do we have nested transformations in React?
Can you also add a test for using nested transformations? 

`<Image><cl-transformation width="100"/></Image>`

What happens when the the props of `c-ltransformation` are updated?

Reply:
@patrick-tolosa such a test already exists here:
https://github.com/cloudinary/cloudinary-react/blob/master/test/TransformationTest.js#L71